### PR TITLE
OLM manifests for 0.0.4

### PR DESCRIPTION
**Pull request by pmorie**

### CODE REVIEW

File: manifests/federation-v2/0.0.4/federation-v2.v0.0.4.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+                      memory: 64Mi
+                  command:
+                    - /root/controller-manager
+                  args:
+                    - --federation-namespace=$(FEDERATION_NAMESPACE)
+                    - --install-crds=false
+                    - --limited-scope=true
+                    - --registry-namespace=$(CLUSTER_REGISTRY_NAMESPACE)
+                  imagePullPolicy: Always
+                  env:
```
Comment (SamiSousa) [inline manifests/federation-v2/0.0.4/federation-v2.v0.0.4.clusterserviceversion.yaml]: Are either of the below equivalent to the `WATCH_NAMESPACE` env variable suggested by OLM?

Comment (pmorie) [reply manifests/federation-v2/0.0.4/federation-v2.v0.0.4.clusterserviceversion.yaml]: Possibly, but not in a way I'm willing to change at this point.

File: manifests/federation-v2/0.0.4/federation-v2.v0.0.4.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,286 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: federation-v2.v0.0.4
+  namespace: placeholder
```
Comment (SamiSousa) [inline manifests/federation-v2/0.0.4/federation-v2.v0.0.4.clusterserviceversion.yaml]: Are there any examples for federation CRs? If so these can be added into `metadata.annotations.alm-examples` as stringified JSON.

Comment (pmorie) [reply manifests/federation-v2/0.0.4/federation-v2.v0.0.4.clusterserviceversion.yaml]: I'll point you at some per our offline convo

File: manifests/federation-v2/0.0.4/federation-v2.v0.0.4.clusterserviceversion.yaml (hunk 1 above)
Comment (pmorie) [reply manifests/federation-v2/0.0.4/federation-v2.v0.0.4.clusterserviceversion.yaml]: Look in the vendored upstream chart for how namespaced vs. not are differentiated

Comment (pmorie) [reply manifests/federation-v2/0.0.4/federation-v2.v0.0.4.clusterserviceversion.yaml]: It's not an exact match for the OLM mechanics yet

**Pull request merged**